server/app/services/geo/ServiceAreaState.java

```diff
@@ -11,6 +14,8 @@ public enum ServiceAreaState {
   // The check failed for technical reasons.
   FAILED("Failed");
 
+  private static final Logger logger = LoggerFactory.getLogger(ServiceAreaState.class);
```

Comment:
**nitpick (blocking):** 

screaming snake case for constants

```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(ServiceAreaState.class);
```

Reply:
updated